filament/backend/src/opengl/PlatformEGL.cpp

```diff
@@ -75,11 +75,15 @@ static void clearGlError() noexcept {
 
 // ---------------------------------------------------------------------------------------------
 
-PlatformEGL::PlatformEGL() noexcept = default;
+PlatformEGL::PlatformEGL(EGLDisplay display = EGL_NO_DISPLAY) noexcept {
```

Comment:
Declare the default parameter value in the header pls

Reply:
780f261